examples/array-map-symmetry/array_map_symmetry.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/io;
+
+public function main() {
+    // List constructors are used to construct list values
+    string[] list = ["foo", "bar"];
+    map<int> 'map = {x: 1, y: 2};
+
+    // Indexing in arrays and maps are used to access elements
+    string listIndex = list[0];
+    io:println(listIndex);
+
+    int? mapIndex = 'map["x"];
+    io:println(mapIndex);
+
+    // Tuple type can use to define per-index member type in array
+    [int, string, boolean] _ = [1, "John", true];
+
+    // Record type can use to define per-index member type in map
+    record {
+        int id;
+        string name;
+        boolean checked;
+    } _ = {id: 1, name: "John", checked: true};
+
+    // Rest type with tuples can use to define open types in tuples
```

Comment:
We don't have a rest type per se. I feel like this doesn't reflect what we want to say.
Can we just say
```suggestion
    // Declares an open type as a tuple type.
```
or improve accordingly. Let's make the same change for mapping as well.